reana_server/config.py

```diff
@@ -389,6 +388,19 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 DEFAULT_WORKSPACE_RETENTION_RULE = "**/*"
 """Workspace retention rule which will be applied to all the workflows by default."""
 
+# Interactive sessions configuration
+# ==================
+_reana_session_max_inactivity_period_env = os.getenv(
+    "REANA_SESSION_MAX_INACTIVITY_PERIOD", "forever"
+)
+if _reana_session_max_inactivity_period_env == "forever":
+    REANA_SESSION_MAX_INACTIVITY_PERIOD: Optional[str] = None
```

Comment:
Small suggestion: perhaps it would be better to keep a long and descriptive name and use `REANA_INTERACTIVE_SESSION` prefix for this value so that it wouldn't be confused to other type of sessions reana might have? WDYT?